src/Roulette.js

```diff
@@ -251,6 +251,62 @@ class Roulette extends React.Component {
                     </label>
                   </div>
                 </form>
+              </Row> */}
+              <Row id="search-row">
+                  <label for="location" id="search-text" >Location</label>
+                  <select class="options" id="location" onChange={this.handleChange} data-testid="location-dropdown">
+                      {/* 50 Most Populated WA Cities (Hardcoded*/}
+                      <option>Select City</option>
```

Comment:
Note for Kenzie: Add this as post-beta task to switch to a component for Google Places API autocomplete